wiki/fr/fr-Server-Linux.md

```diff
@@ -0,0 +1,228 @@
+---
+layout: wiki
+title: "Installation du serveur sous Linux"
+lang: "fr"
+permalink: "/wiki/Server-Linux"
+---
+
+# Installation du serveur sous Linux
+
+
+**_Veuillez vous assurer que vous avez bien lu&nbsp;: [Gestion d'un serveur](Running-a-Server)_**
+
+<a id="running-a-server-with-the-gui"></a>
+## Executer le serveur avec l'interface graphique (_GUI_)
+
+Si vous prévoyez d'exécuter le serveur sur votre machine de bureau (et que vous avez déjà [installé le client Jamulus](Installation-for-Linux)), vous pouvez l'exécuter dans le [mode serveur de votre choix](Choosing-a-Server-Type) en exécutant Jamulus avec l'option `-s` comme suit&nbsp;:
+
+1. Ouvrez un terminal de commande (`CTRL+ALT+t` sur Ubuntu et distributions dérivées).
+1. Partons du principe que Jamulus est dans `/usr/local/bin`, et tapons `Jamulus -s`
+
+Validez avec `[Entrée]` et vous devriez voir la fenêtre de contrôle du serveur. Vous pouvez arrêter le serveur en fermant la fenêtre, ou en tapant `CTRL+C` dans le terminal.
+
+**Pour configurer le serveur**, référez vous aux [instructions pour les utilisateurs de Windows ou Macintosh](Server-Win-Mac).
+
+Voir aussi les [options de la ligne de commande](Command-Line-Options) pour les paramètres que vous pouvez passer.
+
+<a id="running-a-headless-server"></a>
+## Serveur sans interface graphique (_"headless" server_)
+
+Le guide suivant permet de faire fonctionner Jamulus comme un "pur" serveur sur du matériel dédié sans audio ni interface graphique (par exemple un serveur dédié en datacenter ou dans le _cloud_ ou encore une machine dédiée) et suppose des distributions Ubuntu/Debian utilisant systemd. Nous avons également des instructions pour les [Raspberry Pi](Server-Rpi) qui sont bien faites.
+
+* _Un utilisateur de Jamulus, [Grigory](https://sourceforge.net/u/cidnurg/profile/), maintient une **[image Docker pour Jamulus](https://hub.docker.com/r/grundic/jamulus)** que vous pouvez utiliser._
+
+* _Un autre utilisateur de Jamulus, [skrul](https://sourceforge.net/u/skrulx/profile/), à égualement déployé un certain nombre de **public Amazon EC2 AMIs** sous Ubuntu 18.04 pour faire fonctionner Jamulus. [Voir cette page](Jamulus-Cloud) (en anglais) pour plus de détails._
+
+<a id="compiling-source"></a>
```

Comment:
You should link the id of the heading you want to show (this will highlight it automatically).

For "Compiler le code source, création de l'utilisateur `jamulus`" it would probably be something like compiler-le-code-source-creation-de-l-utilisateur-jamulus

Reply:
I've tryed with this, but I don't know why it doesn't works! That's why the HTML anchor.

Reply:
> I've tryed with this, but I don't know why it doesn't works! That's why the HTML anchor.

It doesn't seem to work with accented characters or the single quote.
I will try to fix it by modifying the titles (if possible).

Reply:
Did you build the site locally with jekyll? What does the HTML say? I will also clone your repo and build it.

Reply:
> I will try to fix it by modifying the titles (if possible).

Should be fixed.

Reply:
Ok. I just realized that the id for this heading is "compiler-le-code-source-création-de-lutilisateur-jamulus"